src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+server:
+  port: 8081
+
+spring:
+  h2:
+    console:
+      enabled: true
+
+  profiles:
+    active: test
+
+  datasource:
+    url: jdbc:h2:mem:test
+    driver-class-name: org.h2.Driver
+
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
```

Comment:
Wchodzimy z tym na proda?

Reply:
Oczywiście, że nie 😊 Chciałem, żeby przy każdym uruchomieniu aplikacji dane w DB były odświeżone, gdyby ktoś z Was chciał ponownie odpalić aplikację i przeklikać endpointy. Z tego co mi wiadomo, to na produkcji raczej stosuje się już specjalistyczne narzędzie do zarządzania schematem bazy danych np. `Liquibase`, więc chyba auto-create odpada.